src/scheduler.rs

```diff
@@ -434,6 +434,80 @@ impl Scheduler {
         }
         Ok(())
     }
+
+    /// Send a PayJoin from LND using automatic coin selection and
+    /// automatic fee rate of 2 target confirmations.
+    ///
+    /// `danger_accept_self_signed_uri` should be used for testing only.
+    /// PayJoins are vulnerable to man-in-the-middle attacks, so they must
+    /// communicate over secure connections in production.
+    pub async fn send_payjoin<'a>(
+        &self,
+        uri: bip78::Uri<'_>,
+        danger_accept_self_signed_uri: bool,
+    ) -> Result<Txid, SchedulerError> {
+        log::debug!("get original_psbt");
+        let pj_uri = bip78::UriExt::check_pj_supported(uri)
+            .map_err(|e| SchedulerError::UriDoesNotSupportPayJoin(e.to_string()))?;
```

Comment:
By throwing an error here, we ONLY support to PayJoin `Uri`s. We do not support regular sends. Consider if this is expected behavior or not.

Reply:
Now that we throw an error up to the UI via alert, I think this is acceptable behavior for this iteration.